client/src/app/site/pages/meetings/pages/mediafiles/modules/mediafile-list/components/mediafile-list/mediafile-list.component.ts

```diff
@@ -243,12 +243,14 @@ export class MediafileListComponent extends BaseMeetingListViewComponent<ViewMed
         if (!this.isMultiSelect) {
             this.fileToEdit = file;
             const meetingFileToEdit = file.getMeetingMediafile();
+            let accessGroups = [this.activeMeeting.admin_group_id];
+            if (meetingFileToEdit?.access_group_ids) {
+                accessGroups = [...meetingFileToEdit.access_group_ids];
+            }
```

Comment:
The access_group_id only needs to be pre-filled if there is either a meeting_mediafile for this meeting or if the mediafile has no parent